### ISSUE TITLE
Pioneer probing for the syil LNC6800

### ISSUE THREAD

**Issue by EvanAscendedTech** [toolpath/syil#35]
From my experiences setting up and running these macro programs there seems to be a slight difference in how different brand
probes operate. The examples and youtube videos show the silver cnc probe in use.
When using the pioneer probe M10719 (Probe on) and M10720 (Probe Off) need registered in the control for macro functionality.

**Comment by JustinSGray** [comment]
yes, I agree with that. The macros could be modified to include a parameter in the PROBECONFIG to optionally include the probe-on and probe-off mcodes. 

We'd welcome a PR with those changes.
